sdk/batch/azure-batch/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Release History
 
+## 12.0.0 (2022-01-15)
+
+### Features
+
+- Added property uploadHeaders to `OutputFileBlobContainerDestination`.
+  - Allows users to set custom HTTP headers on resource file uploads.
+  - Array of type HttpHeader (also being added).
+- Added boolean property `allowTaskPreemption` to `JobSpecification`, `CloudJob`, `JobAddParameter`, `JobPatchParameter`, `JobUpdateParameter`
```

Comment:
`allowTaskPreemption` must be a typo. The Pythonic name for this would be `allow_task_preemption`

Reply:
your correct, updated